src/app/search/basic-search/basic-search.component.html

```diff
@@ -12,6 +12,7 @@
       list="suggestions"
       type="text"
       data-cy="basic-search"
+      [(ngModel)]="search.value"
```

Comment:
what's the purpose of these 2 ngModel??

Reply:
I found that it fixes the issue addressed https://ucsc-cgl.atlassian.net/browse/SEAB-2846, so when you click on the close icon the search results are cleared. I am not sure if there is another way in fixing this.

Reply:
the oddity is that this component is using formControl, `#search`, and also ngModel. Try to remove one at least